cypher/cypher-docs/src/docs/dev/glossary.asciidoc

```diff
@@ -25,9 +25,9 @@ This section comprises a glossary of all the keywords -- grouped by category and
 |<<query-call-subquery, CALL {...}>>        | Reading/Writing   | Evaluates a subquery, typically used for post-union processing or aggregations.
 |<<query-create, CREATE>>                    | Writing     |  Create nodes and relationships.
 |<<administration-constraints-syntax, CREATE CONSTRAINT [existence] [IF NOT EXISTS] ON (n:Label) ASSERT n.property IS NOT NULL>>  | Schema   | Create a constraint ensuring that all nodes with a particular label have a certain property.
-|<<administration-constraints-syntax, CREATE CONSTRAINT [node_key] [IF NOT EXISTS] ON (n:Label) ASSERT (n.prop1[, ..., n.propN]) IS NODE KEY [OPTIONS {optionKey: optionValue[, ...]}]>>  |  Schema | Create a constraint ensuring all nodes with a particular label have all the specified properties and that the combination of property values is unique; i.e. ensures existence and uniqueness.
+|<<administration-constraints-syntax, CREATE CONSTRAINT [node_key] [IF NOT EXISTS] ON (n:Label) ASSERT (n.prop1[, ..., n.propN]) IS NODE KEY [OPTIONS {optionKey: optionValue[, ...]}]>>  |  Schema | Create a constraint that ensures that all nodes with a particular label have all the specified properties and that the combination of property values is unique; i.e. ensures existence and uniqueness.
 |<<administration-constraints-syntax, CREATE CONSTRAINT [existence] [IF NOT EXISTS] ON ()-"["r:REL_TYPE"]"-() ASSERT r.property IS NOT NULL>>  | Schema   | Create a constraint ensuring that all relationships with a particular type have a certain property.
-|<<administration-constraints-syntax, CREATE CONSTRAINT [uniqueness] [IF NOT EXISTS] ON (n:Label) ASSERT (n.prop1[, ..., n.propN]) IS UNIQUE [OPTIONS {optionKey: optionValue[, ...]}]>>  |  Schema | Create a constraint ensuring the uniqueness of the combination of node label and property values for a particular property key combination across all nodes.
+|<<administration-constraints-syntax, CREATE CONSTRAINT [uniqueness] [IF NOT EXISTS] ON (n:Label) ASSERT (n.prop1[, ..., n.propN]) IS UNIQUE [OPTIONS {optionKey: optionValue[, ...]}]>>  |  Schema | Create a constraint that ensures that the uniqueness of the combination of node label and property values for a particular property key combination across all nodes.
```

Comment:
Ah, my butterfingers! One too many _that_. 
...Create a constraint that ensures the uniqueness...